README.md

```diff
@@ -113,12 +123,23 @@ where it can be controlled in the device GUI **Apps** tab.
 curl -s https://api.github.com/repos/AxisCommunications/docker-acap/releases/latest | grep "browser_download_url.*Docker_Daemon_.*_<ARCH>\_signed.eap"
 ```
 
-### Securing the Docker ACAP using TLS
+#### Migrating from rootful Docker ACAP application
+
+When migrating from a rootful Docker ACAP application, any version before 2.0, the following is recommended:
+
+- Copy any images that you want to persist from the device to your computer.
+- Stop the Docker ACAP application.
+- Uninstall the Docker ACAP application.
+- If you use the SD card as storage either format it or manually remove the `dockerd` directory (`/var/spool/storage/SD_Disk/dockerd`).
```

Comment:
If you use the SD card as storage either format it or manually remove the `dockerd` directory (`/var/spool/storage/SD_DISK/dockerd`).
```suggestion
- If you use the SD card as storage either format it or manually remove the `dockerd` directory (`/var/spool/storage/SD_Disk/dockerd`).
```